scipy/special/_add_newdocs.py

```diff
@@ -12659,3 +12659,1011 @@ def add_newdoc(name, doc):
     r"""
     Internal function, do not use.
     """)
+
+add_newdoc(
+    "_beta_pdf",
+    """
```

Comment:
```suggestion
    r"""
```

These need to be raw strings. It looks like it’s causing a test failure.